### ISSUE TITLE
higan opens above screen

### ISSUE THREAD

**Issue by darkmage0707077** [higan-emu/higan#150]
higan opens partially "above" the screen area of my window. Different resolutions have different amounts - screenshot is 1280x960 to show most pronounced version. higan can be quickly moved into correct position as a workaround, but it's very annoying that the main menu is above the screen area every time it opens, and I imagine it would be very confusing to new users who aren't aware of how higan operates.

Screen resolution is set to 1366x768 Landscape; changing the resolution doesn't seem to help. Trying to "force" higan to remember its last location using Windows 10 methods also doesn't work; it always opens in the same location. 

![1280_960](https://user-images.githubusercontent.com/30854209/106367996-70ece080-6314-11eb-92be-bf1f4630bba2.png)

**Comment by Screwtapello** [comment]
In the Settings→Scale menu, try setting the smallest scale?

**Comment by darkmage0707077** [comment]
Still off the screen, unfortunately. Not as much, but still enough for the menu bar to be offscreen and require adjustment.

**Comment by Screwtapello** [comment]
Do you have an additional monitor connected? Have you had a monitor connected at some time in the past, that maybe Windows thinks is still hanging around for some reason?

**Comment by darkmage0707077** [comment]
No, only ever the single laptop screen.

**Comment by lowpursuit** [comment]
I’m having the same issue. Laptop is 1920x108 with 125% zoom. When you turn zoom off it’s fine. You can hit ALT + Space then Maximize as a work around

**Comment by Screwtapello** [comment]
Oh, right, high DPI! @darkmage0707077, are you using some high DPI setting too?

**Comment by darkmage0707077** [comment]
Yeah, I thought about that, but it's set to 100% scale.

And I've got numerous workarounds for it; I can also just manually resize a part of the window I'm able to see slightly and it automatically moves the whole thing into view.

But as I said, it's rather cumbersome to have to do this or most other workarounds each time I start higan up, and I'm worried about whether new users might be unduly frustrated by it and be discouraged from trying higan because of it.

**Comment by Screwtapello** [comment]
I agree; unfortunately I don't have any ideas about what might cause this issue; I'll leave it open in case somebody else has some inspiration.

**Comment by kev4cards** [comment]
I have the same issue as darkmage. My maximum resolution is 1366x768, also. No scaling is performed through Windows.

I believe the problem has to do with the total height of the program vs the pixel height of the display. I think higan is only checking either the game display area's or the settings panel's height against the monitor's display height, not both the game display area's and settings panel's heights.
